packages/gestalt/src/Masonry.js

```diff
@@ -29,7 +28,14 @@ type Layout =
   | LegacyMasonryLayout
   | LegacyUniformRowLayout;
 
-type Props<T> = {|
+export type Layer = {|
```

Comment:
I know this is internal, but the naming is a little confusing (seeing as we also have a Layer component in Gestalt). How about reusing `Position` from the layout? They're the same things.

Reply:
done. using Position